packages/babel6/__tests__/babel6.test.js

```diff
@@ -1,15 +1,20 @@
 import test from 'ava'
+import sinon from 'sinon'
 import babel6 from '../index'
 
 test('Babel default options work', (t) => {
   const block = babel6()
 
+  const addLoader = sinon.spy((loaderDef) => config => config)
   const context = {
     fileType: () => '*.js'
   }
 
-  t.deepEqual(block(context), {})
-  t.deepEqual(block.post(context).module.loaders, [
+  t.deepEqual(block(context, { addLoader })({}), {})
+  t.deepEqual(block.post(context, { addLoader })({}), {})
```

Comment:
Right, and it would still be a unit test even if it used the real util functions. That was my point. I'd urge you to remove sinon as a dependency and do that instead (another pull is fine) or port to a test that uses `createConfig` (and still remove sinon). I don't think sinon is a good fit for functional projects and I wouldn't want to encourage contributors to misuse it.